optaplanner-website-root/content/blog/2023/04/05/BoostVRPListSelectorScalability.adoc

```diff
@@ -0,0 +1,76 @@
+= Boosting list variable scalability with nearby selection
+dupliaka
+2023-04-05
+:page-interpolate: true
+:jbake-type: post
+:jbake-tags: insight, design, vehicle routing. move selector
+:jbake-social_media_share_image: nearby-selection.png
+
+As a solver of optimization problems, the VRP ListSelector faces a significant challenge when it comes to scalability.
```

Comment:
Unfortunately, there has not been any article about the list planning variable itself. Thus, this article builds upon something the user cannot be familiar with.

I cannot ask you to write a standalone blog post about the feature, but could you at least provide an introduction to it, referring to the documentation for those who want to learn more?

Nearby selection has been there for the chained model for a long already; this is about closing the gaps between the chained model and the list planning variable.